opentelemetry-sdk/CHANGELOG.md

```diff
@@ -64,6 +64,16 @@
  `LogExporter` trait no longer requires a mutable ref to `self`. If the exporter
  needs to mutate state, it should rely on interior mutability.
  [2764](https://github.com/open-telemetry/opentelemetry-rust/pull/2764)
+- Added Two new methods to the LogRecord struct's public API:
```

Comment:
```suggestion
- Added Two new methods to the `SdkLogRecord` struct's public API:
```

Can you share why we are not adding these new APIs to the `LogRecord` trait itself?

Reply:
> Can you share why we are not adding these new APIs to the LogRecord trait itself?

+1